src/main/resources/application-openshift-metering-worker.yaml

```diff
@@ -7,10 +7,8 @@ rhsm-subscriptions:
     prometheus:
       metric:
         openshift:
-          metricPromQL: >-
-            max(cluster:usage:workload:capacity_physical_cpu_cores:max:5m) without(environment, region)*on(_id)
-            group_right(prometheus) group(subscription_labels{ebs_account='%s', support=~'Premium|Standard|Self-Support|None'})
-            by (_id,ebs_account, support, usage)
+          metricPromQL: group(subscription_labels{ebs_account='%s',support=~'Premium|Standard|Self-Support|None'})by(_id,ebs_account)*on(_id) group_right(ebs_account) cluster:usage:workload:capacity_physical_cpu_cores:min:5m
```

Comment:
Just an observation more than anything else...but sometimes we have application properties using camelCase convention, and other times we use dash-case